KS.Fiks.IO.Arkiv.Client/Schema/dokumentfilHentResultat.xsd

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns="http://www.arkivverket.no/standarder/noark5/dokumentfil/hent/v2"
+	xmlns:xs="http://www.w3.org/2001/XMLSchema"
+	xmlns:n5mdk="http://www.arkivverket.no/standarder/noark5/metadatakatalog/v2"
+	targetNamespace="http://www.arkivverket.no/standarder/noark5/dokumentfil/hent/v2"
+	elementFormDefault="qualified">
+	<xs:import namespace="http://www.arkivverket.no/standarder/noark5/metadatakatalog/v2"
+		schemaLocation="./metadatakatalog.xsd"/>
+
+	<xs:element name="dokumentfil" type="dokumentfil"/>
```

Comment:
Er det en XML som skal returneres for denne tjenesten? Det er vel selve filinnholdet?

Reply:
Nei, det gikk litt fort i svingene her. Glemte selve innholdet :)